index.haml

```diff
@@ -586,15 +586,50 @@
           configEditor.setValue(defaultConfig);
       });
 
+      q('#export-cypher-btn').addEventListener('click', () => {
+
+        let pg = pgParser.parse(editor.getValue());
+        let output = "";
+        pg.nodes.forEach(node => {
+          let node_label = (node.labels[0] === undefined) ? "UNDEFINED" : node.labels[0]
+          let query = "";
+          query = query + "CREATE (v:" + node_label + " {"; // Restriction: single vertex label
+          query = query + "id: '" + node.id + "'"; // ID is stored as a string property
+          for (let entry of Object.entries(node.properties)) {
+            query = query + ", " + entry[0] + ": '" + entry[1] + "'"; // values are always stored as sting
+          }
+          query = query + "})";
+          output = output + query + '\n';
+        });
+        pg.edges.forEach(edge => {
+          let edge_label = (edge.labels[0] === undefined) ? "UNDEFINED" : edge.labels[0]
+          let query = "";
+          query = query + "MATCH (src), (dst) WHERE src.id = '" + edge.from + "' AND dst.id = '" + edge.to + "' ";
+          query = query + "PROPERTIES (";
+          query = query + "e.direction = '" + edge.direction + "'";
+          for (let entry of Object.entries(edge.properties)) {
+            query = query + ", e.\"" + entry[0].toUpperCase() + "\" = '" + entry[1] + "'"; // values are always stored as sting
+          }
+          query = query + ") ";
+          query = query + "FROM MATCH ( (src), (dst) ) ON " + graphNamePGQL + " ";
+          query = query + "WHERE src.id = '" + edge.from + "' AND dst.id = '" + edge.to + "';";
+          output = output + query + '\n';
+        });
+        download(output, 'graph_' + currentTimeString() + '.pgql', 'text/plain');
```

Comment:
Here, `export-cypher-btn` seems not to be added into index.h{a,t}ml. and the extension is still pgql. 
I guess some changes are not included.

Reply:
Sorry, my mistake!